app/controllers/content/node_controller.rb

```diff
@@ -20,9 +20,12 @@ class Content::NodeController < ApplicationController
   layout 'casebooks'
 
   def edit
+    # editing a resource
     if @casebook.public
       return redirect_to details_casebook_path(@casebook)
     end
+    # this breaks the resource view
+    @content = @content.decorate(context: {action_name: action_name, casebook: @casebook, section: @section, context_resource: @resource})
```

Comment:
For some reason this breaks the resource section of this form `app/views/content/edit_details.html.erb` -- nothing in the form changed and as far as i saw the objects were still rendering the same but I must be missing something.

Just some thoughts about that file 
- It's a nested form 
- @resource is a resource object, but @content.resource is the specific resource object
aka @resource.resource  == @content.resource 

@resource -> 
```#<Content::Resource id: 138, title: "Link to google.com", slug: nil, subtitle: "", headnote: "", public: true, casebook_id: 36, ordinals: [5], copy_of_id: nil, is_alias: nil, resource_type: "Default", resource_id: 27, created_at: "2017-08-10 17:46:29", updated_at: "2017-10-18 20:58:59"> ```

@content.resource ->
```#<Default id: 27, name: nil, url: "http://google.com/dfdf", description: nil, public: true, karma: nil, created_at: "2017-08-10 17:46:29", updated_at: "2017-10-18 20:58:59", pushed_from_id: nil, content_type: nil, user_id: 0, ancestry: nil, created_via_import: false>```

The decorator didn't change either of those values